src/hw/raspi4/startup.s

```diff
@@ -0,0 +1,97 @@
+.section ".text.boot"
+
+.globl _Reset
+_Reset:
+  // read cpu id, stop slave cores
+  mrs x1, mpidr_el1
+  and x1, x1, #3
+  cbz x1, 2f
+  // cpu id > 0, stop
+1:wfe
```

Comment:
When using labels please name them, even if the names are seemingly super obvious.

Branching to numbers with the 1f kinda syntax is usually used when names will get repeated. If you can give them textual names do so.

Reply:
Also put labels on their own line. For example here I'd do:
```
wfe:
  wfe
  b wfe
on_core_0:
 <...>
```